modules/couchbase/src/main/java/org/testcontainers/couchbase/CouchbaseContainer.java

```diff
@@ -85,6 +85,16 @@
 
     private final List<BucketDefinition> buckets = new ArrayList<>();
 
+    private Integer dataMemoryQuota;
```

Comment:
What about storing it in a Map<CouchbaseService, Integer> instead? Then potentially we can extend it in the future and also need only one setter. The actual implementation can then just iterate the map.